docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -25,9 +25,10 @@ _Versions: XX.XX.XX, YY.YY.YY, ZZ.ZZ.ZZ_
 == Upgrading to v8.15.x may cause ingestion to fail
 
 _Elastic Stack versions: 8.15.0+_
+_Fixed in Elastic Stack version 8.15.4_
```

Comment:
Two minor suggestions:

* Be more specific about the affected versions now that there's a fix.
* Break into two lines (using a trailing ` +` on the first line):

| Before | After |
|---|---|
|  <img width="811" alt="Screenshot 2024-11-08 at 9 45 10 AM" src="https://github.com/user-attachments/assets/22d73135-b6c5-4171-96f3-6b980771b777"> | <img width="725" alt="image" src="https://github.com/user-attachments/assets/6515ab89-4420-48c7-92dd-ef5ac16f1bcd"> |

```suggestion
_Elastic Stack versions: 8.15.0, 8.15.1, 8.15.2, 8.15.3_ +
_Fixed in Elastic Stack version 8.15.4_
```